### ISSUE TITLE
Error on dependency node-pre-gyp, downloads from remote

### ISSUE THREAD

**Issue by gudbrand3** [kelektiv/node.bcrypt.js#827]
Hi,

Getting and error when doing npm install on bcrypt for dependency node-pre-gyp which is sort of an error but maybe more a warning as it resolves to using https remote download which works fine.Any way to get this resolved or changed to a warning/info/something instead of an error as it causes the CI/CD pipeline to fail but isnt really a error as it works?

```
bcrypt@5.0.0 install /apps/dev/node_modules/bcrypt
node-pre-gyp install --fallback-to-build
[error]node-pre-gyp
[error]WARN
[error]Using request for node-pre-gyp https download
[bcrypt] Success: "/apps/dev/node_modules/bcrypt/lib/binding/napi-v3/bcrypt_lib.node" is installed via remote
```

**Comment by agathver** [comment]
Doesn't look like bcrypt is causing CI to fail. We don't print "error" but
we do write to stderr as warning.

How does your CI check for error? If bcrypt successfully installs it should
exit normally.

**Comment by gudbrand3** [comment]
Im Using Azure DevOps Pipeline and Releases. I tested now with a Pipeline build instead of direct releases pipeline with SSH task. In Pipeline as part of building the artifact all is ok = success

When I do it directly (no artifact build) using release pipeline and the SSH task basically a basic npm install cmd it is recognizing the output WARN as a "failure" which then fails the release pipeline = failure

I tested just from my mac with npm install in my IDE (webstorm) and it comes as a WARN but still "ok", so not recognized as a failure here. I dont know, might just be DevOps SSH task which is annoying. 

> bcrypt@5.0.0 install /Users/username/project/node_modules/bcrypt
> node-pre-gyp install --fallback-to-build

node-pre-gyp **WARN** Using request for node-pre-gyp https download 
[bcrypt] Success: "/Users/username/project/node_modules/bcrypt/lib/binding/napi-v3/bcrypt_lib.node" is installed via remote

I guess it is the WARN log type it dosnt like
